### PR TITLE
DBZ-4715 Fix stream event's source timestamp tz resolution

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-4715

### CODE REVIEW

File: debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleStreamingChangeEventSourceMetrics.java [hunk 1]
```diff
@@ -694,6 +694,10 @@ public void calculateTimeDifference(OffsetDateTime databaseSystemTime) {
         LOGGER.trace("Current time {} ms, database difference {} ms", now.toEpochMilli(), timeDiffMillis);
     }
 
+    public long getDatabaseOffsetSeconds() {
```
Comment (gunnarmorling) [inline debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleStreamingChangeEventSourceMetrics.java:697]: Can we use a more semantical type for representing `offsetSeconds`, e.g. `java.time.ZoneOffset`?

**Review by gunnarmorling — COMMENTED** [review]
One comment inline.

**Review by gunnarmorling — APPROVED** [review]
LGTM, merging. Thx, @Naros!

**Comment by marun790** [comment]
Hi @gunnarmorling 
Do we have any release date for this fix to be used in our application.

**Comment by gunnarmorling** [comment]
Yes, this will be part of 1.9.0.Beta1, which is scheduled for early March.

**Comment by marun790** [comment]
awesome!!, thanks @gunnarmorling .

**Pull request merged**